samples/instances.js

```diff
@@ -15,38 +15,202 @@
 
 'use strict';
 
-function listInstances() {
-  // [START bigtable_list_instances]
-  // Imports the Google Cloud client library
-  const Bigtable = require('@google-cloud/bigtable');
+// Imports the Google Cloud client library
+const Bigtable = require('@google-cloud/bigtable');
 
-  // Creates a client
+async function runInstanceOperations(instanceName, clusterName, clusterName2) {
   const bigtable = new Bigtable();
 
-  // Lists all instances in the current project
-  bigtable
-    .getInstances()
-    .then(results => {
-      const instances = results[0];
-      console.log('Instances:');
-      instances.forEach(instance => console.log(instance.id));
-    })
-    .catch(err => {
-      console.error('ERROR:', err);
+  console.log(`\nCreating a PRODUCTION Instance`);
```

Comment:
Don't use a newline on console.log to have empty line, use an empty call to `console.log()` instead